src/PointwiseFunctions/AnalyticData/ForceFree/RotatingDipole.hpp

```diff
@@ -0,0 +1,202 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/ForceFree/Tags.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticData/AnalyticData.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace ForceFree::AnalyticData {
+
+/*!
+ * \brief The magnetosphere of an isolated rotating star with dipolar initial
+ * magnetic field in the flat spacetime. This is a toy model of a pulsar
+ * magnetosphere.
+ *
+ * \note Radius of the star is rescaled to 1.0
+ *
+ * The vector potential of the initial magnetic field has the form
+ * \cite Most2022
+ *
+ * \begin{equation}
+ *  A_\phi = \frac{A_0 \varpi_0 (x^2+y^2)}{(r^2 + \delta^2)^{3/2}}
+ * \end{equation}
+ *
+ * where $A_0$ is the vector potential amplitude, $\varpi_0$ is a constant with
+ * the unit of length, $r^2 = x^2 + y^2 + z^2$, and $\delta$ is a small number
+ * for regularization of the dipole magnetic field at the origin ($r=0$).
+ *
+ * In the Cartesian coordinates, components of densitized magnetic fields are
+ * given as
+ *
+ * \begin{align}
+ *  \tilde{B}^x & = A_0 \varpi_0 \frac{3xz}{(r^2 + \delta^2)^{5/2}} , \\
+ *  \tilde{B}^y & = A_0 \varpi_0 \frac{3yz}{(r^2 + \delta^2)^{5/2}} , \\
+ *  \tilde{B}^z & = A_0 \varpi_0 \frac{3z^2 - r^2 + 2\delta^2}{(r^2 +
+ *                    \delta^2)^{5/2}} .
+ * \end{align}
+ *
+ * Electric fields and charge density are set to zero everywhere.
+ *
+ * Rotation of the star is switched on at $t=0$ with the angular velocity
+ * specified in the input file. Inside the star we impose the MHD condition
+ * $\mathbf{E} + \mathbf{v} \times \mathbf{B} = 0$, where the velocity field is
+ * given by $\mathbf{v} \equiv \Omega \hat{z} \times \mathbf{r}$. (see docs of
+ * the mutator ImposeMhdConditionInsideNs). By this means, initial dipolar
```

Comment:
This is to be added in a future PR, correct?  (Not a change request, just making sure that's why I can't find it.)